apps/dialtone-documentation/docs/guides/contributing/index.md

```diff
@@ -21,10 +21,10 @@ If you need to add an icon into Dialtone, here’s how you would go about doing
 
 ### For system icons
 
-1. Create a new branch in [dialtone-icons] repo starting with "feat/" in the name.
-2. Place the exported SVG file(s) in the appropriate folder category inside `./src/svg/`
+1. Create a new branch starting with "feat/" in the name.
+2. Place the exported SVG file(s) in the appropriate folder category inside `packages/dialtone-icons/src/svg/`
 3. If you need to add keywords related to the icon.
-   - Add the icon name to `./src/icons.json` into the correct category.
+   - Add the icon name to `packages/dialtone-icons/src/icons.json` into the correct category.
```

Comment:
This file is autogenerated so this is not needed any more

Reply:
This should be pointing to `keywords.json`, thanks for finding it!